portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -229,15 +227,65 @@ function allowUserSubmit() {
       });
 }
 
-/* inserts a functioning searchbar into the navigation bar of a page. */
-function insertSearch() {
+/**
+ * Gets the other html pages as documents and invokes method to insert Search
+ * bar
+ */
+async function loadSearch() {
+  const otherDocs =
+      await Promise.all([getHTML('/about.html'), getHTML('/statistics.html')]);
```

Comment:
what about if you load this on the statistics page? how will it get the html from index.html?

Reply:
My goal is to have it working for the index page before making it work for every page.